### ISSUE TITLE
Removes uv sets

### ISSUE THREAD

**Issue by stickypig** [wjakob/instant-meshes#23]
When I import an obj, it removes all of the uv sets I previously had, making it unusable if I ever want to apply high poly textures to it that contained multiple UV sets.

**Comment by wjakob** [comment]
Yes, this is a current limitation of the software. I am actually curious: is it possible to have multiple sets of UV coordinates in a single OBJ file? Would you mind sharing such for a file, so that we can at some point figure out how to parse and integrate that?

**Comment by ssh4net** [comment]
stickypig just create new good optimised UV on retopo model. and bike textures from HR model to retopo model. For example in xNormal.

wjakob. HR models rarely have good UV. Mostly this is scan with worst UV possible.
Better is your app will create new optimised UV than tried protect UV from HR model. 
Because no problem at all, bike textures from HR to LR models.
But no UV on retopo model required open it in other 3D app for UV creation.

**Comment by stickypig** [comment]
I’m sorry I’m not understanding what you’re saying. I don’t have a bike model. I understand high poly models don’t have good UV’s, thats why I’m trying to retopo, but it still needs to maintain the material ID data that was previously in there. Your application should not be ‘removing’ meta data from the model

Drew Andersen | Lead Designer | StickyPig| dribbble.com/stickypig. | M: +1 801 419 7848 [tel:+1%20801%20419%207848]
On Sun, Apr 24, 2016 at 10:36 AM, ssh4net <notifications@github.com> wrote:
stickypig just create new good optimised UV on retopo model. and bike textures from HR model to retopo model. For example in xNormal.

wjakob. HR models rarely have good UV. Mostly this is scan with worst UV possible.
Better is your app will create new optimised UV than tried protect UV from HR model.
Because no problem at all, bike textures from HR to LR models.
But no UV on retopo model required open it in other 3D app for UV creation.

**Comment by ssh4net** [comment]
Stickypig, Sorry, misprint. I mean "Bake" not "bike".

i don't seen commercial software that "protect" UV from Highres model.
All software bake Highres data to new lowpoly. Because retopology completely retopologise all, vertex, poly and as is normals, And new model required new UV.

So, for example, just create new UV on new model in blender, or in other 3D software. And bake textures from Highres model to new lowres model. For this you can use xCode for example.

**Comment by yosun** [comment]
why not include a texture re-baker?
